fuzzing/fuzz/src/parser.rs

```diff
@@ -36,7 +39,50 @@ impl<'a> super::Scenario<'a> for Scenario<'a> {
 
     fn run(&self) -> Result<(), Self::RunError> {
         let Scenario { syntax, src } = self;
-        Ast::from_str(src, None, syntax)?;
+        let _: Ast<'_> = Ast::from_str(src, None, syntax)?;
         Ok(())
     }
 }
+
+impl fmt::Display for Scenario<'_> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        let Scenario { syntax, src } = self;
+        let syntax = if *syntax == Syntax::default() {
+            Cow::Borrowed("Syntax::default()")
+        } else {
+            let InnerSyntax {
+                block_start,
+                block_end,
+                expr_start,
+                expr_end,
+                comment_start,
+                comment_end,
+            } = **syntax;
+            Cow::Owned(format!(
+                "\
+SyntaxBuilder {{
+        name: \"test\",
```

Comment:
The indent is a bit broken here. ;)

Reply:
It's actually intentional. The `syntax` should be indented by two tabs, except on the first line, where it comes directly after `let syntax = `.